benchmarks/gravity_thin_shell/gravity_thin_shell.prm

```diff
@@ -0,0 +1,82 @@
+# A simple setup for testing the gravity postprocessor using the 
+# sample scheme 'map'.
```

Comment:
Benchmarks prms ideally have descriptions at the beginning that describe what the prm is benchmarking, how it is doing that, what the numerical comparisons are and why it is useful. See, for example `benchmarks/viscoelastic_bending_beam/viscoelastic_bending_beam.prm` It would be good to extend this blurb by a few lines to cover these details.

Reply:
done